tests/test_xero_bookmarks.py

```diff
@@ -72,3 +72,29 @@ def test_run(self):
         self.check_offsets(bookmarks)
         self.look_for_unexpected_bookmarks(bookmarks)
         self.assertIsNone(state.get("currently_syncing"))
+
+
+class TestArchivedContacts(XeroBookmarks):
+
+    def get_properties(self):
+        properties = super().get_properties()
+        properties["includeArchivedContacts"] = "true"
+
+        return properties
+
+
+    def test_get_archived_contacts(self):
+        runner.run_check_job_and_check_status(self)
+
+        found_catalogs = menagerie.get_catalogs(self.conn_id)
+        self.check_all_streams_in_catalogs(found_catalogs)
+        # Replicating the contacts stream
+        self.select_specific_catalog(found_catalogs, "contacts")
+
+        # clear state and run the actual sync
+        menagerie.set_state(self.conn_id, {})
+        runner.run_sync_job_and_check_status(self)
+        records = runner.get_upserts_from_target_output()
+
+        contacts_status = [record["ContactStatus"] for record in records]
+        self.assertIn("ARCHIVED", contacts_status)
```

Comment:
It would be better if we could assert every status is `"ARCHIVED"`, rather than one. I find using a set comprehension to collect values and using `assertSetEqual` to be the cleanest way to check a specific value across records.

Reply:
This comment can be ignored, I misunderstood the state of our test data. Not all contacts should be archived.